apps/src/codebridge/FileTabs/FileTabs.tsx

```diff
@@ -66,6 +70,13 @@ export const FileTabs = React.memo(() => {
     }
   }
 
+  function handleTabActivation(event: React.KeyboardEvent, fileId: string) {
+    if (event.key === 'Enter' || event.key === ' ') {
+      // We don't stop event propagation here because we want the close button to work.
```

Comment:
*Not doing a stop propagation is only an issue if there are extra listeners waiting on an enter or spacebar right? Just making sure I'm thinking through other listeners that might activate if this goes outside this scope (but I can't think of anything)

Reply:
I think so? In this case there is, it's the close button, which is why I'm not doing a stop propagation. I didn't see any unexpected issues when testing, but there could always be a corner case.